src/testdir/test_cmdline.vim

```diff
@@ -3444,4 +3444,18 @@ func Test_setcmdline()
   cunmap a
 endfunc
 
+func Test_cmdhistadd()
+  messages clear
+
+  call assert_equal(execute('messages'), '')
```

Comment:
The order of arguments are `assert_equal({expected}, {actual})`.

Reply:
and I think should use [`GetMessages()`](https://github.com/vim/vim/blob/master/src/testdir/shared.vim#L385-L394) than `execute('message')` directly since the former will drop the maintainer entry.

Reply:
Fixed.